docs/server-side/v2/Destinations/rtmp-streaming-and-browser-recording.mdx

```diff
@@ -73,11 +73,13 @@ curl \
 
 This is a URL which lands into the meeting directly. If you are using 100ms apps then this will be like - `https://<domain>.app.100ms.live/preview/<room_id>/<role>?skip_preview=true`.
 
-There are also a few additional query params which can be used for the 100ms app link -
+Please check the following to understand how these params are useful while starting RTMP streaming or browser recording with 100ms app links:
 
--   `auth_token=token1234`: this can be used to pass auth token generated on your side to the recording web-app
--   `ui_mode=activespeaker`: this can be used to open active speaker mode by default for recording. In active speaker mode,
-    the current speaker will show up in center, and rest of the peers will show up in the sidebar.
+-   `skip_preview=true`: skips the preview page and directly joins the meeting with header and controls disabled.
+-   `skip_preview_headful=true`: skips the preview page and directly joins the meeting with header and controls enabled.
+-   `auth_token=token1234`: Use this to pass auth token generated on your side to the recording web-app
+-   `ui_mode=activespeaker`: opens active speaker mode by default for recording. In active speaker mode, the current speaker will show up in the center, and rest of the peers will show up in the sidebar.
+-   'name=johndoe': Use this to add a name for the respective peer.
```

Comment:
these quotes are different? not backticks?

Reply:
fixed